films/services.py

```
@@ -34,7 +32,26 @@ def process(self):
                     new_film.save()
             except:
                 pass
-        return True
+
+    @staticmethod
+    def adding_genres_in_db_from_list(list_genres):
+        for genre in list_genres:
+            try:
+                new_genre = Genre(name=genre)
+                new_genre.save()
+            except:
+                pass
```

except: pass не ок, нет конкретизации что упадает и почему